<?xml version="1.0"?>
<launch>

	<node name="stairsdetection" pkg="ros_stairsdetection" type="stairsdetection" output="screen">
		<param name="input" value="/camera/depth/points" />
		<param name="steps" value="/hmmwv/steps" />
		<param name="stairways" value="/hmmwv/stairways" />
		<param name="camera_frame" value="camera_link" />
		<param name="robot_frame" value="base_link" />
		<param name="world_frame" value="map" />
		<param name="namespace" value="hmmwv" />
		<param name="publish_steps" value="true" />
		<param name="publish_stairways" value="true" />

		<param name="segmentation_iterations" value="1000" />
		<param name="segmentation_threshold" value="0.01" />
		<param name="min_step_width" value="0.5" />
		<param name="max_step_width" value="1.0" />
		<param name="min_step_height" value="0.15" />
		<param name="max_step_height" value="0.23" />
	</node>

</launch>
